### DISCUSSION TITLE
Background

### DISCUSSION THREAD

**Discussion by fronderblash** [rrirower/DP2SceneGraphWizard#27] [Q&A]
Hello  
Thanks for this great tool 
Is there a way to make the background look like this 

https://github.com/rrirower/DP2SceneGraphWizard/assets/141512200/370e275b-7a1f-408e-9443-a2911c4d573c

**Comment by rrirower** [comment]
Possibly.  I would need to look into it further. I'll mark this as an enhancement.

**Comment by fronderblash** [comment]
Okay thanks

**Comment by fronderblash** [comment]
Hello 
Have you find a way to do the background

**Reply by rrirower** [reply]
As I've already indicated, this has been marked as an enhancement and does not have a high priority on the list of remaining tasks.  It will be looked into when time allows.

**Reply by rrirower** [reply]
This is implemented in alpha-8.

**Reply by fronderblash** [reply]
Okay
Alpha 8 have background like this right 

https://github.com/rrirower/DP2SceneGraphWizard/assets/141512200/7ca78575-8ff9-4b0a-9889-b940be1e6a93

**Reply by ferdiworks** [reply]
That would be nice

**Reply by rrirower** [reply]
@fronderblash you'll need to try it to see if it meets your needs.
